erpnext_com/www/docs/user/manual/en/accounts/pricing-rule.md

```diff
@@ -44,31 +51,46 @@ To have Discount and Price List Rate for an Item auto-applied, create Pricing Ru
 
     <img alt="Discount" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-discount.png">
 
-### 3. Scenarios with Pricing Rule
-#### 3.1 Make Price List
+1. Discount can be set in terms of amount.
+
+    <img alt="Discount" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-discount-amt.png">
+
+### 2.2 Product Discount
+
+1. Buy 2 quantities get 1 free quantity of the same item, to configure such type of rules. Set the Price or Product Discount as Product discount and enable Same Item and set the quantity.
+
+    <img alt="Discount" class="screenshot" src="{{docs_base_url}}/assets/img/articles/pricing-rule-same-product-free.png">
+
+1. Buy 2 quantities get 1 free quantity of the another item, to configure such type of rules. Set the Price or Product Discount as Product discount, disable Same Item and set the Free Item and quantity.
```

Comment:
"Buy 2 quantities get 1 free quantity of the another item." To configure such type of rules. Set the Price or Product Discount as Product discount, untick the Same Item checkbox and set the Free Item and quantity.